haunted-houses/haunted-houses-api/src/main/java/cz/muni/fi/pa165/hauntedhouses/facade/GameInstanceFacade.java

```diff
@@ -7,10 +7,23 @@
  * @author Zoltan Fridrich
  */
 public interface GameInstanceFacade {
-
+    /**
+     * Finds the GameInstance of given Player
+     * @param playerId ID of the Player
+     * @return GameInstanceDTO of given Player if he exists and has a GameInstance, null otherwise
+     */
     GameInstanceDTO findGameInstanceByPlayerId(Long playerId);
 
+    /**
+     * Creates new GameInstance
+     * @param gameInstance DTO containing all information that is required for creating new GameInstance
+     * @return ID of the GameInstance
+     */
     Long createGameInstance(GameInstanceCreateDTO gameInstance);
```

Comment:
Tu by to chcelo dopísať, čo sa stane ak create spôsobí nejaký constraint violation. To isté aj pre update a aj v services

Reply:
Opraveno